### PR TITLE
switch to minimal-snowplow-tracker

### PR DESCRIPTION
**Pull request by cmcarthur**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ PyYAML>=3.11
 psycopg2==2.7.5
 sqlparse==0.2.3
 networkx==1.11
-snowplow-tracker==0.7.2
+minimal-snowplow-tracker==0.0.1
```
Comment (cmcarthur) [inline requirements.txt]: remove other pinned dependencies because of snowplow-tracker (esp celery)

update setup.py, too

File: requirements.txt [hunk 2]
```diff
@@ -5,7 +5,7 @@ PyYAML>=3.11
 psycopg2==2.7.5
```
Comment (cmcarthur) [inline requirements.txt]: >=2.7.5, <2.8

Comment (cmcarthur) [reply requirements.txt]: remove pytz

Comment (cmcarthur) [reply requirements.txt]: possibly remove argparse?

File: setup.py [hunk 3]
```diff
@@ -39,13 +39,11 @@ def read(fname):
...
         'PyYAML>=3.11',
-        'psycopg2==2.7.5',
+        'psycopg2>=2.7.5,<2.8',
         'sqlparse==0.2.3',
         'networkx==1.11',
-        'snowplow-tracker==0.7.2',
-        'celery==3.1.23',
+        'minimal-snowplow-tracker==0.0.1',
         'snowflake-connector-python>=1.4.9',
         'requests>=2.18.0',
```
Comment (beckjake) [inline setup.py]: This should probably be `>=2.18.0,<3` - requests is very good about semver but that means any `3.0` will be pretty much guaranteed to break something...

**Pull request merged**